### PR TITLE
math-magicians: events

### PR DESCRIPTION
**Pull request by tugsem**
## In this branch I :
- Copied the files calculate.js and operate.js into a logic/ directory in my project,
- Imported the files in the Calculator component,
- Implemented the event handlers.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,66 +1,58 @@
...
                     </tr>
                     <tr>
-                    <td><input className='button' type='button' value='7' onClick= {this.display('7')}/></td>
-                    <td><input className='button' type='button' value='8' onClick= {this.display('8')}/></td> 
-                    <td><input className='button' type='button' value='9' onClick= {this.display('9')}/></td> 
-                    <td><input className='button orange' type='button' value='X' onClick= {this.display('X')}/></td> 
+                    <td><input className='button' type='button' value={7} onClick= {this.handleClick}/></td>
+                    <td><input className='button' type='button' value={8} onClick= {this.handleClick}/></td> 
+                    <td><input className='button' type='button' value={9} onClick= {this.handleClick}/></td> 
+                    <td><input className='button orange' type='button' value='x' onClick= {this.handleClick}/></td>
```
Comment (elisha2kyakpo1) [inline src/components/Calculator.js:34]: - [ ] Kindly fix the error in the image below. The error only comes when I click the multiplication sign.

math_error

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @tugsem, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors 👍
- Descriptive commit messages 👍
- The PR title and description look good and professional 👍 

## Required Changes ♻️
_Check the comments under the review._

- [x] Kindly update and customize the readme file, you can use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) from Microverse.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (tugsem) [reply src/components/Calculator.js:34]: @elisha2kyakpo1  Hello, I don't see that issue on my browser. So don't know how to fix. I used 'x' as a value for that button, but you see 'X' in the console. I get confused. 😕

Comment (elisha2kyakpo1) [reply src/components/Calculator.js:34]: @tugsem I used chrome, you can try deleting browser history and run the project again.

**Review by Emmyn5600 — APPROVED** [review]
Hi  @tugsem 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada: :clap: :clap: :100:

nodding-yes

## Optional suggestions

- [ ] n/a

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**